toolkit/offchain/src/reserve/update_settings.rs

```diff
@@ -64,7 +174,7 @@ fn update_reserve_settings_tx(
 			.with_plutus_data(&(datum.clone().into()))
 			.with_script_ref(&ScriptRef::new_plutus_script(&reserve.scripts.validator.to_csl()))
 			.next()?;
-		let mut val = reserve.validator_version_utxo.value.to_csl()?;
+		let mut val = reserve_utxo.value.to_csl()?;
```

Comment:
Please change  `// mint new settings` above (can leave comment there), because there is no mint. Only output is defined.

Reply:
Also, I was able to remove `.with_script_ref(&ScriptRef::new_plutus_script(&reserve.scripts.validator.to_csl()))` and tests are still passing